src/main/java/com/avispl/symphony/dal/avdevices/wirelesspresentation/mersive/solsticepodgen3/SolsticePodGen3Communicator.java

```diff
@@ -521,6 +592,18 @@ protected void authenticate() {
 		//Solstice Pod doesn't require API token.
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	protected HttpHeaders putExtraRequestHeaders(HttpMethod httpMethod, String uri, HttpHeaders headers) throws Exception {
+		if (isFormUrlEncodedRequest) {
```

Comment:
is it possible to figure this out based on the requested URL? So we won't need to keep the state in the class variable